source/en/carbon_edge.rst

```diff
@@ -202,24 +202,11 @@ Interpreting Results
 
 Final Results
 ~~~~~~~~~~~~~
--  **Parameter log**. Each time the model is run, a text (.txt) file
-      will appear in the \ *Workspace*. The file will list the parameter
-      values for that run and will be named according to the service,
-      the date and time. When contacting NatCap about errors in a model
-      run, please include the parameter log.
-
--  **carbon_map_[Suffix].tif**. A map of carbon stock per pixel, with
-      the amount in forest derived from the regression based on distance
-      to forest edge, and the amount in non-forest classes according to
-      the biophysical table. Note that because the map displays carbon
-      (Mg) per pixel, coarser resolution maps should have higher values
-      for carbon, because the pixel areas are larger.
-
--  **aggregated_carbon_stocks_[Suffix].shp**: If service areas of
-      interest are provided by the user, this output shapefile
-      summarizes the total carbon (column *c_sum*) and mean carbon per
-      ha (column *c_ha_mean*) in the area polygons defined. Values are
-      in Mg.
+-  **Parameter log**. Each time the model is run, a text (.txt) file will appear in the *Workspace*. The file will list the parameter values for that run and will be named according to the service, the date and time. When contacting NatCap about errors in a model run, please include the parameter log.

```

Comment:
```suggestion
-  **Parameter log**. Each time the model is run, a text (.txt) file will appear in the *Workspace*. The file will list the parameter values for that run and will be named according to the service, date, and time. Please include the parameter log when contacting NatCap about errors in a model run.

```

Reply:
Added Oxford comma and switched from passive to active voice, per the [UG style guide](https://docs.google.com/document/d/1BHwHDu_I-x0s_2GsbUb4rfVmXMkl7kl97sx2suBTLh8/edit?usp=sharing).